README.md

```diff
@@ -9,6 +9,30 @@ An abstraction layer for low-level platform-specific graphics APIs.
 * Reference documentation: http://wiki.gpfault.net/docs/nicegraf
 * Sample code: https://github.com/nicebyte/nicegraf/tree/master/samples
 
+# prerequisites
+
+## Windows
+
+- CMake 3.14.0 or above
+- Visual Studio 2022 with v143 build tools (found in Desktop Development with C++ section)
+
+*Optional*:
+- VulkanSDK (for debug purposes)
+
+# running the samples
+
+## Windows
+
+- Run <b>build-samples.sh</b> script
+- Open **.sln** from **sample-build-files** and build the solution
+- Open **samples/binaries** and run the samples
+
+# faq
```

Comment:
let's remove this section for now. instead, add "make sure cmake is within system path"  to the build steps.